doc/source/dev/reviewer_guidelines.rst

```diff
@@ -0,0 +1,135 @@
+  #123" where 123 is the issue number.
+- For code changes, at least one core team member (those with commit rights)
+  should review and approve all pull requests. If you are the first to review a
+  PR and approve of the changes use the GitHub `approve review
+  <https://help.github.com/articles/reviewing-changes-in-pull-requests/>`_ tool
+  to mark it as such. If you are a subsequent reviewer, please approve the
+  review or, if you have commit rights and think no more review is needed, merge
+  the PR.
+
+Core team members
+-----------------
+  
+- Make sure all automated CI tests pass before merging a PR, and that the
+  :ref:`documentation builds<building-docs>` without any errors.
+- In case of merge conflicts, ask the PR submitter to :ref:`rebase on master
+  <rebasing-on-master>`.
+- For PRs that add new features or are in some way complex, wait at least a day
+  or two before merging it. That way, others get a chance to comment before the
+  code goes in. Consider adding it to the release notes.
+- When merging contributions, a committer is responsible for ensuring that those
+  meet the requirements outlined in the :ref:`Development process guidelines
+  <guidelines>` for NumPy. Also, check that new features and backwards
+  compatibility breaks were discussed on the `numpy-discussion mailing list
+  <https://mail.python.org/mailman/listinfo/numpy-discussion>`_.
+- Backports and trivial additions to finish a PR (really trivial, like a typo or
+  PEP8 fix) can be pushed directly.
+- Squashing commits or cleaning up commit messages of a PR that you consider too
+  messy is OK. Remember to retain the original author's name when doing this.
+  Make sure commit messages follow the :ref:`rules for NumPy
+  <writing-the-commit-message>`.
+- When you want to reject a PR: if it's very obvious, you can just close it and
+  explain why. If it's not, then it's a good idea to first explain why you
+  think the PR is not suitable for inclusion in NumPy and then let a second
+  committer comment or close.
+
+GitHub Workflow
+---------------
+
+When reviewing pull requests, please use workflow tracking features on GitHub as
+appropriate:
+
+- After you have finished reviewing, if you want to ask for the submitter to
+  make changes, change your review status to "Changes requested." This can be
+  done on GitHub, PR page, Files changed tab, Review changes (button on the top
+  right).
+- If you're happy about the current status, mark the pull request as Approved
+  (same way as Changes requested). Alternatively (for core team members): merge
+  the pull request, if you think it is ready to be merged.
+
+It may be helpful to have a copy of the pull request code checked out on your
+own machine so that you can play with it locally. One way to do this is to
+navigate to the NumPy root directory in the terminal and enter::
+  
+  git fetch upstream pull/PULL_REQUEST_ID/head:NEW_BRANCH_NAME
```

Comment:
Let's replace this bit with `gh` or `hub` CLI usage (or get rid of it).

Reply:
Oh wait, that's below. I think that's so far superior that we should just delete this old hacky method.

Reply:
this apparently does work, learned something. resolving.